sklearn/linear_model/_ridge.py

```diff
@@ -2394,12 +2395,12 @@ class RidgeCV(MultiOutputMixin, RegressorMixin, _BaseRidgeCV):
         (i.e. data is expected to be centered).
 
     scoring : str, callable, default=None
-        A string (see model evaluation documentation) or
-        a scorer callable object / function with signature
-        ``scorer(estimator, X, y)``.
-        If None, the negative mean squared error if cv is 'auto' or None
-        (i.e. when using leave-one-out cross-validation), and r2 score
-        otherwise.
+        A string (see `model evaluation documentation
+        <https://scikit-learn.org/stable/modules/model_evaluation.html#scoring-parameter>`_)
```

Comment:
We should never use links to our own docs. The links can change, we should instead use rst's references, for instance:

```
`model evaluation documentation <scoring_parameter>`
```